.gitignore

```diff
@@ -1,6 +1,8 @@
 /project/
 flakes
 logs/
+*.html
```

Comment:
I don't thing adding *.html is a good idea. Better just stick with server/jacoco

Reply:
done.